src/app/styles/components/_preview-card.scss

```diff
@@ -55,6 +56,7 @@
         flex-flow: column nowrap;
         align-items: stretch;
         justify-content: space-between;
+        margin-bottom: 5rem;
```

Comment:
Makes sense to me. Unfortunately `$home-section-padding` is in px and that value is used in a bunch of calculations elsewhere as px. So I guess this stays as is for now.